src/views/HomePage.js

```diff
@@ -1 +1,16 @@
 //This is a file for the home page
+import React from "react";
+import { Title } from "../components/Title/Title";
+import { CohortList } from "../components/CohortList/CohortList";
+import { LineGraph } from "../components/Graphs/LineGraph";
+import {HomePagePageStyle} from "./HomePageStyle";
```

Comment:
The compoment name HomePagePageStyle seems a little redundant to me. I think it can be "HomePageStyle".

Reply:
Please check the updated pull request =]